samples/captioning/multi-track-captions.html

```diff
@@ -67,6 +70,7 @@
     <body onload="startVideo()">
         <div class="dash-video-player">
             <video controls="true"></video>
+            <div id="ttml-rendering-div"></div>
```

Comment:
@AkamaiDASH. I agree that it may make simpler to compare if there are two files which only differ in this aspect. It should be clear from the naming that this is the only difference, so it should maybe include something like style-rendering in the name.